### PR TITLE
[C2341] Delete an existing match profile (folijet) (TaaS)

### PR DESCRIPTION
**Pull request by AliaksandrShuleika**
[testrail](https://foliotest.testrail.io/index.php?/cases/view/2341)
[report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4753/allure/#suites/0f29a4212ddbc0a40cf0646f9bcb6758/eea8ef3eb6563053/)

### CODE REVIEW

File: cypress/support/fragments/data_import/match_profiles/modals/exceptionDelete.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Button, Modal } from '../../../../../../interactors';
+
+const exceptionDeleteModal = Modal({ id: 'delete-matchProfiles-exception-modal' });
+
+export default {
+  closeException: () => cy.do(exceptionDeleteModal.find(Button('Close')).click()),
+  verifyExceptionMessage: () => cy.expect(
+    exceptionDeleteModal.has({
+      message:
+          'This match profile cannot be deleted, as it is in use by one or more job profiles.',
```
Comment (ernestyedigaryan1) [inline cypress/support/fragments/data_import/match_profiles/modals/exceptionDelete.js:10]: The exception message in the test case is different, please clarify this part with the functional QA engineer

**Review by ernestyedigaryan1 — APPROVED** [review]

File: cypress/support/fragments/data_import/match_profiles/modals/exceptionDelete.js (hunk 1 above)
Comment (AliaksandrShuleika) [reply cypress/support/fragments/data_import/match_profiles/modals/exceptionDelete.js:10]: could you please point where you can see exception message in the test case?

File: cypress/e2e/data-import/settings/delete-an-existing-match-profile.cy.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+      createMappingProfile: `autotest mappingProfileForCreate${getRandomPostfix()}`,
+      createMatchProfile: `autotest matchProfileForCreate${getRandomPostfix()}`,
+    };
+
+    const calloutMessage = `The match profile "${matchProfileToDelete.profileName}" was successfully deleted`;
+
+    before('Create test data', () => {
+      cy.createTempUser([Permissions.settingsDataImportEnabled.gui]).then((userProperties) => {
+        user = userProperties;
+        NewFieldMappingProfile.createMappingProfileViaApi(profile.createMappingProfile).then(
```
Comment (TetianaParanich) [inline cypress/e2e/data-import/settings/delete-an-existing-match-profile.cy.js:45]: You don't need to create mapping and action profiles. It's enough only match and job profile

File: cypress/support/fragments/data_import/match_profiles/modals/exceptionDelete.js (hunk 1 above)
Comment (ernestyedigaryan1) [reply cypress/support/fragments/data_import/match_profiles/modals/exceptionDelete.js:10]: sure, in the step 3
You will be presented with a "Cannot delete match profile" prompt and your only option will be to click "Close" and return to the Match profile view

**Review by TatianaOne — APPROVED** [review]

**Pull request merged**